docs/Other SDL Features/Remote Control Vehicle Features/index.md

```diff
@@ -309,67 +618,37 @@ sdlManager.sendRPC(buttonPress);
 ```
 !@
 
-### Subscribing to Changes
-It is also possible to subscribe to changes in data associated with supported modules. To do so, during your request for data, simply set `subscribe` to `true`. To unsubscribe, send the request again with `subscribe` set to `false`. The response to a subscription will come in a form of a notification. You can receive this notification by adding a notification listener for `OnInteriorVehicleData`.
+@![java,javaEE,javaSE]
+```java
+// ToDo - Add example
+```
+!@
 
-!!! NOTE
-The notification listener should be added before sending the @![iOS]`SDLGetInteriorVehicleData`!@ @![android, javaSE, javaEE]`GetInteriorVehicleData`!@ request.
-!!!
+#### Releasing the Module (RPC v6.0+)
+When the user no longer needs control over a module, you should release the module so other users can control it. If you do not release the module, other users who would otherwise be able to control the module may be rejected from doing so.
```

Comment:
```suggestion
When the user no longer needs control over a module you should release the module so it can be used by other users. If you do not release the module, other users who have permission to control the module may be rejected when attempting to take control of the module.
```

Reply:
I prefer the original wording and grammar.